prov/efa/src/rxr/rxr_read.h

```diff
@@ -38,22 +38,43 @@
 #ifndef _RXR_RDMA_H_
 #define _RXR_RDMA_H_
 
+/*
+ * read can used in 2 scenarios:
+ *
+ * 1. application posted a read request.
+ *
+ * 2. read message protocol is being used, receiver is going
+ *    to post a read requst.
+ *
+ * To distinguish them, we use a pointer as context.
```

Comment:
This isn't needed, there is already a type field in rxr_tx_entry/rxr_rx_entry.

Reply:
Yes, it is not needed now. This is mainly preparation for the future when we are adding a case packet entry used as context. Because ```rxr_pkt_entry``` has to remain 64 bytes, we cannot add any new members to it.

Reply:
I see - I hadn't read the last commit before making this comment.